resolvers/simple.resolver.js

```diff
@@ -22,16 +22,19 @@ module.exports = async (job, log, data) => {
 
   allScripts.forEach((script) => {
     let scriptContents = log.substr(log.indexOf(script));
-    scriptContents = scriptContents.split('\n').slice(1).join('\n');
-    scriptContents = scriptContents.substr(0, scriptContents.indexOf('" exited with ')).trim();
-    scriptContents = scriptContents.split('\n').slice(0, -1).join('\n');
-    scriptContents = scriptContents.trim();
+    const exitCode = /exited\swith\s(\d)/g.exec(scriptContents);
+    if (Number(exitCode[1]) !== 0) {
```

Comment:
Also, could you add logging on this? require the logger at the top of the file, using: `const logger = require('../utils/logger');`, and then use `logger.log` to log what was the exit code for command x (where x is the name if the script)

Reply:
Sure.